spring-rabbit/src/test/java/org/springframework/amqp/rabbit/connection/LocalizedQueueConnectionFactoryIntegrationTests.java

```diff
@@ -63,6 +63,7 @@ public void testConnect() {
 		RabbitTemplate template = new RabbitTemplate(targetConnectionFactory);
 		template.convertAndSend("", queue.getName(), "foo");
 		assertEquals("foo", template.receiveAndConvert(queue.getName()));
+		admin.deleteQueue(queue.getName());
```

Comment:
Because they previously were not auto-delete so running against a broker that has previously had the tests run will fail.

We can remove these in a couple of years or so 😄